pxr/usd/usdPhysics/parseUtils.cpp

```diff
@@ -0,0 +1,3124 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/pxr.h"
+
+#include "pxr/base/gf/transform.h"
+
+#include "pxr/usd/usdGeom/gprim.h"
+#include "pxr/usd/usdGeom/mesh.h"
+#include "pxr/usd/usdGeom/sphere.h"
+#include "pxr/usd/usdGeom/cube.h"
+#include "pxr/usd/usdGeom/capsule.h"
+#include "pxr/usd/usdGeom/cylinder.h"
+#include "pxr/usd/usdGeom/cone.h"
+#include "pxr/usd/usdGeom/plane.h"
+#include "pxr/usd/usdGeom/points.h"
+
+#include "pxr/usd/usdGeom/metrics.h"
+
+#include "pxr/usd/usdShade/materialBindingAPI.h"
+
+#include "parseUtils.h"
+#include "parseDesc.h"
+
+#include "pxr/usd/usdPhysics/scene.h"
+#include "pxr/usd/usdPhysics/collisionGroup.h"
+#include "pxr/usd/usdPhysics/materialAPI.h"
+#include "pxr/usd/usdPhysics/rigidBodyAPI.h"
+#include "pxr/usd/usdPhysics/collisionAPI.h"
+#include "pxr/usd/usdPhysics/articulationRootAPI.h"
+
+#include "pxr/usd/usdPhysics/joint.h"
+#include "pxr/usd/usdPhysics/fixedJoint.h"
+#include "pxr/usd/usdPhysics/revoluteJoint.h"
+#include "pxr/usd/usdPhysics/prismaticJoint.h"
+#include "pxr/usd/usdPhysics/sphericalJoint.h"
+#include "pxr/usd/usdPhysics/distanceJoint.h"
+
+#include "pxr/usd/usdPhysics/limitAPI.h"
+#include "pxr/usd/usdPhysics/driveAPI.h"
+#include "pxr/usd/usdPhysics/filteredPairsAPI.h"
+#include "pxr/usd/usdPhysics/meshCollisionAPI.h"
+
+
+#include "pxr/usd/usdGeom/pointInstancer.h"
+
+#include "pxr/base/work/dispatcher.h"
+#include "pxr/base/work/loops.h"
+
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+void ParseFilteredPairs(const pxr::UsdPrim& usdPrim, pxr::SdfPathVector& filteredPairs)
```

Comment:
Looks like "pxr::" namespace also needs to be cleaned from here. As mentioned before this code is within the PXR_NAMESPACE_OPEN_SCOPE code guard and will appropriately handle pxr::. It fails to build in some configs and internally as well because of the explicit "pxr::" namespace.